uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -325,7 +401,7 @@ define([
      * @return {Promise}
      */
     function initBalanceChange () {
-      return (vm.isMode('edit') ? getOriginalBalanceChange() : calculateBalanceChange());
+      return (!vm.isMode('create') ? getOriginalBalanceChange() : attemptCalculateBalanceChange());
```

Comment:
why this change from `vm.isMode('edit')` to `vm.isMode('create')` ?

Reply:
This is described in the PR description. This is because we want to fetch the original balance breakdown in "view" mode as well.